### ISSUE TITLE
Is Incompatible With Aldeed:Collection2

### ISSUE THREAD

**Issue by davidjaenisch** [alethes/meteor-pages#146]
This library is not compatible with the SetSchema function of Aldeed:Collection2.

**Comment by gokhanbaydar** [comment]
Yes, it is not compatible, and aldeed:autform 5.3.0 does not work if this package is used in the project. 

Uncaught TypeError: Cannot read property 'renderFunction' of undefined at pages.coffee: 473

and 

Exception in template helper: TypeError: Cannot read property 'schema' of undefined
    at Object.getDefs (http://localhost:3000/packages/aldeed_autoform.js?8a3a72752639b5186173691898c4ac267bd8d888:255:18)
    at Object.autoFormGetComponentContext [as getComponentContext] (http://localhost:3000/packages/aldeed_autoform.js?8a3a72752639b5186173691898c4ac267bd8d888:516:20)
    at Object.afQuickFieldIsGroup (http://localhost:3000/packages/aldeed_autoform.js?8a3a72752639b5186173691898c4ac267bd8d888:7358:30)
    at bindDataContext (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2880:16)
    at Blaze._wrapCatchingExceptions (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:1651:16)
    at http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2928:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:3476:12)
    at wrapHelper (http://localhost:3000/packages/blaze.js?a5c324925e5f6e800a4c618d71caf2848b53bf51:2927:27)
    at Object.Spacebars.call (http://localhost:3000/packages/spacebars.js?7bafbe05ec09b6bbb6a3b276537e4995ab298a2f:172:18)
    at http://localhost:3000/packages/aldeed_autoform.js?8a3a72752639b5186173691898c4ac267bd8d888:7309:22

**Comment by wesleyfsmith** [comment]
You sure? I'm using collection2 with this package just fine.

**Comment by hkirk** [comment]
@wesleyfsmith could you tell who you managed it?
